# COMP472MP3

current things about the code
- I initially dpwngraded to python 3.8 because the gensim documentation asked for it, and the assignment also asked for it.
- Trying to use the gensim.download.load with this constantly gave me errors saying "unable to read local cache during fallback, connect to Internet and retry"
- Some research about the error told me that gensim does not support python 3.8 and that I needed to downgrade more, so I downgraded to python 3.6 which the posts say should be supported by the function we are trying to use, but I am still having the same error...
- At this point, the synonyms.csv file we are going to be using is added to the repo and is easy to access once we get the gensim functions working...
- Let me know if you make any progress, otherwise I will discuss it with my TA on monday, she may be able to get us on a right direction